packages/graphql/schemas/schema.graphql

```diff
@@ -65,6 +65,12 @@ type CachedUser implements Node {
   id: ID!
 }
 
+"""Feature not available for subscription"""
```

Comment:
Ditto here with the schema changes

Reply:
Yeah committed here since this must have been left out of another PR in error, no need for us all to keep seeing these